tests/providers/cncf/kubernetes/operators/test_spark_kubernetes.py

```diff
@@ -570,7 +571,9 @@ def test_resolve_application_file_template_file(dag_maker, tmp_path, session):
     (tmp_path / filename).write_text("foo: {{ ds }}\nbar: {{ dag_run.dag_id }}\nspam: egg")
 
     with dag_maker(
-        dag_id="test_resolve_application_file_template_file", template_searchpath=tmp_path.as_posix()
+        dag_id="test_resolve_application_file_template_file",
+        template_searchpath=tmp_path.as_posix(),
+        session=session,
```

Comment:
In all other cases we fixes tests for database isolation we used `serialized=True` as parameter for dag_maker to ensure changes are persisted to DB.

Reply:
...but tests are green - good to go :-D

Reply:
I overlooked that part. My bad! I am happy to see they have passed without it. Thanks for the review and approval. :)